### ISSUE TITLE
RGB OLED: Font Loader

### ISSUE THREAD

**Issue by CarsonGracey** [parallaxinc/BlocklyProp#422]
This block will be special for the OLED as it has no connections and must be loaded on its own into EEPROM.

![font loader](https://cloud.githubusercontent.com/assets/20099916/17066860/cb2528de-4ffd-11e6-91d6-7b4333deea00.PNG)

It will generate this code and download the library (which is still being developed):

```C
#include "oledc_fontLoader.h"

int main()
{
  oledc_fontLoader();
}
```

Block factory link: https://blockly-demo.appspot.com/static/demos/blockfactory/index.html#x8dk9t
Once this block is loaded into EEPROM it can then be deleted from the project so all fonts will be accessible at any time.

**Comment by kgracey** [comment]
OK, I understand how this works and we can certainly document the OLED use this way: run a program with this block and then create a new code to do what you want with the OLEDs. This block just lives "on his own" so to speak.

**Comment by MatzElectronics** [comment]
Just sent the library needed for this in, but there is one small but important note:

The #include generated by the block needed to contain an underscore in the name.  Please see below:
![image](https://cloud.githubusercontent.com/assets/19737272/17265200/dedbf200-55a1-11e6-9348-1688352fecab.png)

**Comment by CarsonGracey** [comment]
When this block is the only one in the project (as it should be) there is an extra space at line 8 and the L in 

```C
#include "oledc_fontLoader.h"
```

needs to be capitalized since simple libraries are case sensitive.

**Comment by zfi** [comment]
Corrected in PR #505.

**Comment by CarsonGracey** [comment]
Tested and Verified.
